tests/infra/node.py

```diff
@@ -88,27 +88,34 @@ def __init__(
         self.suspended = False
         self.node_id = None
         self.node_client_host = None
+        self.interfaces = []
         self.version = version
         self.consensus = None
 
-        if host.startswith("local://"):
+        if isinstance(host, str):
+            host = infra.e2e_args.HostSpec.from_str(host)
+
+        if type(host) == str:
```

Comment:
lines 95 will already do this for all subclasses of str, this is presumably intended to be removed?

Reply:
Good spot, don't know I ended up with both. I'll blame a bad merge.